spring-context/src/test/java/org/springframework/format/datetime/standard/InstantFormatterTests.java

```diff
@@ -121,5 +132,19 @@ Stream<?> provideArguments() {
 					.map(DateTimeFormatter.RFC_1123_DATE_TIME.withZone(systemDefault())::format);
 		}
 	}
+	private static final class RandomEpochMillisProvider implements ArgumentsProvider {
+
+		private static final long DATA_SET_SIZE = 10;
+
+		private static final Random random = new Random();
+
+		@Override
+		public Stream<Arguments> provideArguments(ExtensionContext context) {
+			return random.longs(DATA_SET_SIZE, Instant.ofEpochMilli(Long.MIN_VALUE).toEpochMilli(), Instant.ofEpochMilli(Long.MAX_VALUE).toEpochMilli())
```

Comment:
this can be simplified:
```suggestion
			return random.longs(DATA_SET_SIZE, Long.MIN_VALUE, Long.MAX_VALUE)
```